sdk/tests/js/tests/tex-image-and-sub-image-2d-with-image.js

```diff
@@ -43,7 +43,7 @@ function generateTest(pixelFormat, pixelType, pathToTestRoot, prologue) {
         gl.clearColor(0,0,0,1);
         gl.clearDepth(1);
 
-        wtu.loadTexture(gl, "/resources/red-green.png", runTest);
+        wtu.loadTexture(gl, "../../resources/red-green.png", runTest);
```

Comment:
Do these URLs really change? I thought they were relative to the containing HTML file. The relative URLs are getting a little confusing.